src/main/java/org/example/persistence/ormanager/ORManagerImpl.java

```diff
@@ -28,4 +28,22 @@ static DataSource createDataSource(String url, String user, String password, Map
         System.out.println(connection.isValid(1000));
         return dataSource;
     }
+    @Override
+    public Student save(Student student){
+        Connection connection;
+        try {
+           connection = dataSource.getConnection();
+            Statement statement = connection.createStatement();
+            statement.execute("CREATE TABLE IF NOT EXISTS STUDENTS (first_name VARCHAR(255)) ;");
```

Comment:
About this row and two rows below (row 39): it is always considered good practice to make the sql statements (or the most often used ones) constants and pass a constant in the methods, not the whole string itself.

And about the table creation: right now you are creating a table with just one column. Try checking out the table schema file in the resources directory.